charts/alfresco-sync-service/templates/ingress.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: {{ template "common.capabilities.ingress.apiVersion" . }}
 kind: Ingress
 metadata:
-  name: {{ template "alfresco-sync-service.fullname" . }}-ingress
+  name: {{ template "alfresco-sync-service.fullname" . }}
   annotations:
     kubernetes.io/ingress.class: "nginx"
```

Comment:
check the helm install logs, I am switching to `ingressClassName: nginx` in search service PR